### PR TITLE
ISO639: Converting to text template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

REF: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/iso639

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @tantalor

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/ISO639.pm [hunk 1]
```diff
@@ -12,11 +12,15 @@ zci is_cached => 1;
 
 handle remainder => sub {
     my ($lang, $code) = langpair(shift) or return;
-    return 'ISO 639: '. $lang .' - '. $code,
-    structured_answer => {
-        input => [$lang],
-        operation => 'ISO 639 Language code',
-        result => ($code),
+
+    return 'ISO 639: '. $lang .' - '. $code, structured_answer => {
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/ISO639.pm]: Why not `"ISO 639: $lang - $code"`?

File: lib/DDG/Goodie/ISO639.pm [hunk 2]
```diff
@@ -12,11 +12,15 @@ zci is_cached => 1;
...
+        data => {
+            title => $code,
+            subtitle => "ISO 639 Language code: " . html_enc($lang)
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/ISO639.pm]: Do we really need to `html_enc` this, it's just a word?

**Comment by GuiltyDolphin** [comment]
@mintsoft Thanks! :dolphin:

**Pull request merged**